specs/engage-digital_openapi3.yaml

```diff
@@ -4414,6 +4416,42 @@ paths:
       summary: Get all connected agents status
       tags:
         - Agent Status
+  /surveys:
+    get:
+      description: >-
+        This method renders all surveys ordered by creation date (ascending).
+      operationId: getAllSurveys
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/getAllSurveysResponse'
+          description: Success
+      summary: Getting all surveys
+      tags:
+        - Surveys
+  '/surveys/{surveyId}':
+    get:
+      description: >-
+        This method renders a survey from given id.

```

Comment:
```suggestion
        This method renders a survey for a given id.
```

Reply:
Actually I'm realizing that all other descriptions for get routes are `from given id`, I'll keep it like this for consistency